### PR TITLE
scripts: remove `DMS_DEBUG`

### PR DESCRIPTION
**Pull request by georglauterbach**
# Description

Removes `DMS_DEBUG` as this variable is no longer used in scripts.

During the removal, I noticed that VS Code complained about the bug issue template. I corrected the three lines, hope that's fine.

## Type of change

- [x] Improvement (non-breaking change that does improve existing functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (README.md or the documentation under `docs/`)
- [x] If necessary I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: docs/content/config/environment.md [hunk 1]
```diff
@@ -13,13 +13,6 @@ title: Environment Variables
 - **empty** => uses the `hostname` command to get canonical hostname for `docker-mailserver` to use.
 - => Specify a fully-qualified domainname to serve mail for. This is used for many of the config features so if you can't set your hostname (_eg: you're in a container platform that doesn't let you_) specify it via this environment variable. It will take priority over `docker run` options: `--hostname` and `--domainname`, or `docker-compose.yml` config equivalents: `hostname:` and `domainname:`.
 
-##### DMS_DEBUG
-
-This environment variable is deprecated. Use `LOG_LEVEL` instead.
-
-- **0** => Debug disabled
-- 1     => Enables debug on startup
-
```
Comment (wernerfred) [inline docs/content/config/environment.md:22]: As the old var is referenced in a huge amount of discussions and issue comments we should think about keeping it in here for a few more releases. "is deprecated" should be "was removed" as deprecated would mean it is still in place.

File: mailserver.env [hunk 2]
```diff
@@ -13,11 +13,6 @@
 # => Specify a fully-qualified domainname to serve mail for.  This is used for many of the config features so if you can't set your hostname (e.g. you're in a container platform that doesn't let you) specify it in this environment variable.
 OVERRIDE_HOSTNAME=
 
-# (deprecated: use LOG_LEVEL instead)
-# 0 => Debug disabled
-# 1 => Enables debug on startup
-DMS_DEBUG=0
```
Comment (wernerfred) [inline mailserver.env:19]: Same here

**Review by polarathene — APPROVED** [review]

File: docs/content/config/environment.md (hunk 1 above)
Comment (polarathene) [reply docs/content/config/environment.md:22]: That's a good point! :)

I think I've had that confusing experience as a user before a few times :sweat_smile:

Comment (georglauterbach) [reply docs/content/config/environment.md:22]: Good point, I will take care :D

Comment (georglauterbach) [reply docs/content/config/environment.md]: Done in 05f483554433eec6beadccc076a5432f8e262d38

File: mailserver.env (hunk 2 above)
Comment (georglauterbach) [reply mailserver.env]: Done in 05f483554433eec6beadccc076a5432f8e262d38

**Review by wernerfred — APPROVED** [review]

**Review by casperklein — APPROVED** [review]

**Pull request merged**